### ISSUE TITLE
exec: unrecognised option '-type'

### ISSUE THREAD

**Issue by robertmagier** [OpenZeppelin/solidity-docgen#6]
Hi, 

When I execute command

`solidity-docgen .\smartcontract\ .\smartcontract\contracts\ .\smartcontract\docs\
`

I get this error message

```
exec: unrecognised option '-type'

C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\lib\index.js:133
    throw new Error(['Command line operation failed with code ' + commandOutput.code + '.', 'Standard error output: ' + commandOutput.stderr].join('\n'));
    ^

Error: Command line operation failed with code 1.
Standard error output: unrecognised option '-type'

    at handleErrorCode (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\lib\index.js:133:11)
    at parseProject (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\lib\index.js:123:3)
    at exports.default (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\lib\index.js:20:23)
    at Command.program.version.arguments.option.action (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\bin\solidity-docgen:32:5)
    at Command.listener (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\node_modules\commander\index.js:315:8)
    at Command.emit (events.js:159:13)
    at Command.parseArgs (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\node_modules\commander\index.js:653:12)
    at Command.parse (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\node_modules\commander\index.js:474:21)
    at Object.<anonymous> (C:\Users\wild\AppData\Roaming\npm\node_modules\solidity-docgen\bin\solidity-docgen:44:9)
    at Module._compile (module.js:660:30)
```

Repository I am trying to run it on is here:
[https://github.com/robertmagier/Viddo](https://github.com/robertmagier/Viddo)

Any ideas ? Thank you.

**Comment by FriendlyUser** [comment]
I am running into the same problem on Windows, I think it has to do with windows because I got solidity-docgen working on OSX.

In addition, your project configuration shouldn't matter unless you are importing smart contract packages (openzeppelin contracts).

If you have a linux or mac, I highly recommend testing it on those systems.

**Comment by robertmagier** [comment]
Unfortunately I don't have Linux or Mac. Just Windows. Somehow I can't get any documentation solidity generator running. Doxity also fails with truffle.

**Comment by FriendlyUser** [comment]
@robertmagier, you could try using a tool like [solmd](https://www.npmjs.com/package/solmd) or [socs](https://www.npmjs.com/package/socs) to create markdown-based documentation per contract and then build with documentation generation tools such as [docusaurus](https://docusaurus.io/) or vuepress.

**Comment by ismaelbej** [comment]
It is using a sub shell to collect the *.sol files in that will fail on windows https://github.com/OpenZeppelin/solidity-docgen/blob/f665fd4dae98924f7a0e3154da34327baa1b7b2f/lib/index.js#L122

**Comment by frangio** [comment]
Thanks for reporting @robertmagier! This is the same issue as #22. We've released v0.2 in alpha which fixes this issue. Please see #23 to learn more about the new version.
